MAINTENANCE.md

```diff
@@ -0,0 +1,48 @@
+# Maintainers
+
+The current maintainers can be seen in the [CODEOWNERS](CODEOWNERS) file.
+
+## Responsibilities
+
+Maintainers are responsible for the health of the project. They keep the project
+healthy by:
+
+- Acting as stewards for the community of users
+- Responding to discussions, issues, and pull requests in a timely manner
+- Ensuring changes don’t break existing projects as much as possible (e.g.
+  maintaining Bazel, rules, and Xcode version backwards compatibility)
+- Participating in idea and design discussions
+- Adding support for new Bazel, rules (e.g. rules_apple and rules_swift), and
```

Comment:
That's fine. It wasn't meant to be an exhaustive list (used `e.g.` instead of `i.e.`), but we do list `rules_ios` in the README, so I'm good with being explicit about it here as well.